README.rst

```diff
@@ -326,6 +340,12 @@ You can mutate the candidate payload by passing in a function that accepts one p
 
 Pass a callback function to this method to configure custom grouping logic. The callback should take one parameter, an instance of RaygunMessage, and return a string between 1 and 100 characters in length (see 'Custom Grouping Logic' below for more details).
 
+.. code:: python
```

Comment:
what does the `..` mean?

Reply:
That's the formatting of `rst` files to introduce codeblocks.